scorecard/product_pages.py

```diff
@@ -0,0 +1,98 @@
+import requests
+from .models import BusinessUnit, BusinessUnitGroup, Person, Product, ProductSecurityCapability, ProductSecurityRole, \
+    SecurityCapability, SecurityRole, Status
+from scorecard import scoring
+
+'''
+    Acquires all business unit and product data via the Product Pages API.
+'''
+
+PRODUCT_PAGES_API = 'https://pp.engineering.redhat.com/pp/api/latest'
+BUSINESS_UNITS_API = PRODUCT_PAGES_API + '/bugroups/'
+PRODUCTS_API = PRODUCT_PAGES_API + '/products/'
+SECURITY_CHAMPION = "Security Escalation Contact"
+PROGRAM_MANAGER = "Program Manager"
+PRODUCT_SECURITY = "Product Security"
+PRODUCT_SECURITY_PROGRAM_MANAGER = PRODUCT_SECURITY + " " + PROGRAM_MANAGER
+TRUSTED_CAS = '/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt'
+
+
+def update_product_data():
+
+    # Get all the business unit data and create or update accordingly.
+
+    bu_groups = requests.get(BUSINESS_UNITS_API, headers=dict(Accept='application/json'), verify=TRUSTED_CAS).json()
+    for bu_group in bu_groups:
+        updated_bu_group, _ = BusinessUnitGroup.objects.get_or_create(pp_id=bu_group['id'])
+        updated_bu_group.name = bu_group['name']
+        updated_bu_group.save()
```

Comment:
There's nothing wrong with these three lines, but if you wanted you could simplify down to one DB call by including the non-unique-key fields in the 'defaults' param of get_or_create:
`updated_bu_group, _ = BusinessUnitGroup.objects.get_or_create(pp_id=bu_group['id'], defaults={'name': bu_group['name']})`

Similarly for the other code below.